technical-vision.md

```diff
@@ -7,3 +7,66 @@ We envision a future where participants in the open source ecosystem use and sha
 - Community members can provide information and notifications about product defects, mitigations, quality, and supportability and have this information rapidly flow forward across the ecosystem system to all users, and users can rapidly update their software or implement mitigations as appropriate.
 
 The Open Source Security Foundation (OpenSSF) provides tools, services, training, infrastructure, and resources to achieve this vision.
+
+---
+
+# OpenSSF Technical Vision (v2 proposed)
+
+> _Note: the bullet points that are included below in block quotes would not be included in the final version of the vision, but are listed for reference/discussion._
+
+An aspirational view of the future we seek to create in 2-5 years:
+
+### OpenSSF is a well-respected strategic partner to both large OSS foundations and individual OSS projects
```

Comment:
The OSSF has played a significant role in public policy discussions this year, which I think has been a success. Should we call those out too?

Reply:
Yes, I see that under the 2nd top-level bullet under advocacy.

Reply:
Ah ok, I could see it in there too. I was thinking that we'd be a trusted strategic partner for policy folks to reach out to for guidance, but we can also advocate proactively.

Reply:
I couldn't find the advocacy part, but adding "and policy decision makers" seems fine to me.